suite-native/config/src/enabledNetworks.ts

```diff
@@ -20,4 +22,11 @@ export const mainnetsOrder: NetworkSymbol[] = [
 
 export const testnetsOrder: NetworkSymbol[] = ['test', 'regtest', 'tgor', 'trop', 'tada', 'txrp'];
 
-export const enabledNetworks: NetworkSymbol[] = Object.keys(networks) as NetworkSymbol[];
+export const enabledNetworks: NetworkSymbol[] = Object.keys(networks).filter(
+    network => !deprecatedNetworks.includes(network),
+) as NetworkSymbol[];
+
+export const getEnabledMainnets = () =>
```

Comment:
Why are these functions? They has no params so they are basically constants.

Reply:
No reason. I replaced it with variables instead of functions f271e00c617fa4ee6083eefb70bec7e4607248a4